website/frontend/templates/docs/scripting.html

```diff
@@ -105,7 +105,10 @@ <h3 id="num_num_len">$num(num,len)</h3>
         <ul>
           <li>Returns
             <code>num</code>formatted to
-            <code>len</code>digits.</li>
+            <code>len</code>digits,
```

Comment:
That should be changed back to a period (they're independent clauses), or you should insert "where," like `len digits, where len cannot [...]`.